src/frontend/efiling-frontend/src/modules/errorRedirect.js

```diff
@@ -0,0 +1,16 @@
+export function errorRedirect(errorUrl, error) {
+  if (
+    errorUrl &&
+    error.response &&
+    error.response.status &&
+    error.response.data &&
+    error.response.data.message
+  ) {
+    return window.open(
+      `${errorUrl}?status=${error.response.status}&message=${error.response.data.message}`,
+      "_self"
+    );
+  }
+
+  return null;
```

Comment:
do you think we can add a test case for this function testing the redirect/return of null?

Reply:
Should be covered now.